### ISSUE TITLE
[Suggestion] Add connecting textures support

### ISSUE THREAD

**Issue by lucaszischka** [the-pink-hacker/Classic-3D#17]
**What would you like me to change and or reconsider?**
It would be great if you could add support for connecting textures (preferable lambdabettergrass) especially regarding grass blocks. If you have any questions regarding my request please don't hesitate to ask. Anyways amazing texture pack, a ton of thanks for your work!

**Comment by RyanGar46** [comment]
I looked over the documentation of connected textures with optifine and did not see any way to be able to change the model. It appears that it only works with textures and not models.

[Connected Texture Docs](https://github.com/sp614x/optifine/blob/master/OptiFineDoc/doc/ctm.properties)

**Comment by lucaszischka** [comment]
Yes I had a quick look too now and i see that you can’t make it work with optifine. I also looked into the [lambdabettergrass documentation](https://github.com/LambdAurora/LambdaBetterGrass/tree/1.19/documentation) and it doesn’t seem to be possible there either. However I suggested this advancement because connected textures are quite buggy for me (no connected grass, but instead black pixels). Therefore any solution would be acceptable; here are my ideas:
1. I could open a feature request on optifine and/or lambdabettergrass to allow for models to be changed and not only textures.
2. You could add support without 3d/model (only for the connected sides of the grass block), so that the connected side is not 3d but at least isn’t buggy. However I’m not a texturepack creator and don’t know if this is possible.
